tests/adapters/test_magicc.py

```diff
@@ -0,0 +1,13 @@
+# Example file, would look something like this
```

Comment:
Let’s have this in the docs and only include magicc tests with the magicc adapter later